src/endpoints/http/collections/export/index.js

```diff
@@ -87,6 +87,35 @@ function exportZip(zipFile, out) {
   });
 }
 
+function isValidExportSize(db, collectionNames) {
+  const sizeLimit = fhconfig.value('sizeLimit');
+  if (!sizeLimit) {
+    return Promise.resolve(true);
+  }
+
+  return getCollectionsSize(db, collectionNames)
+    .then(collectionsSize => collectionsSize <= sizeLimit);
+}
+
+/**
+ * Create a human readable size with units from bytes.
+ *
+ * @param {Number} bytes - The size in bytes.
+ * @returns {String} - Human readable size with units.
+ */
+function formatBytes(bytes) {
+  if (bytes === 0) {
+    return '0 Bytes';
+  }
+
+  const k = 1000;
+  const dm = 2;
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
Are you sure that Yottabyte is enough? What if someones database is larger than one trillion terrabytes?